modules/23_capitalMarket/debt_limit/equations.gms

```diff
@@ -23,7 +22,6 @@ q23_limit_surplus_growth(t,regi)..
   + sum(tradePe, (pm_pvp(t,tradePe)/(pm_pvp(t,"good")+0.000000001))*(vm_Mport(t,regi,tradePe)- vm_Xport(t,regi,tradePe))) 
   + (pm_pvp(t,"perm")/(pm_pvp(t,"good")+0.000000001)) * (vm_Mport(t,regi,"perm") - vm_Xport(t,regi,"perm"))
   + sum(tradeSe, pm_MPortsPrice(t,regi,tradeSe) * vm_Mport(t,regi,tradeSe)) - sum(tradeSe, pm_XPortsPrice(t,regi,tradeSe) * vm_Xport(t,regi,tradeSe)) 
-  + vm_budgetTradeM(t,regi) - vm_budgetTradeX(t,regi)
```

Comment:
did you discuss this with Marian?

Reply:
These are budget terms I previously added when I first merged my initial draft of the capacity realisation. I didn't discuss adding them or removing them now with anyone, because so far they are all set to zero in all realisations except my new one.

In other words: this is always zero in `standard` trade, so it doesn't matter.